source/blog/20019-01-22-sso-keycloak-ovirt-openidc-389ds.html.md

```diff
@@ -0,0 +1,267 @@
+---
+title: oVirt OIDC with keycloak using 389ds user federation
```

Comment:
I'd simplify it to `Federate oVirt autthetication to OpenID Connect infrastructure`